src/components/SignIn/index.tsx

```diff
@@ -94,6 +105,11 @@ const SignIn: React.FC<SignInProps> = ({
         [handleChangeFocusField]
     );
 
+    const isButtonDisabled = (): boolean => {
+        return (((captchaType === 'recaptcha' && !reCaptchaSuccess) ||
+        (captchaType === 'geetest' && !geetestCaptchaSuccess)) && captchaLogin()) ? true : false;
```

Comment:
if u will combine reCaptchaSuccess && geetestCaptchaSuccess please update this place too